README.md

```diff
@@ -181,3 +182,40 @@ any outputs.
 | `name`    | Unique name for this rule (required)                                                |
 | `srcs`    | Sass files included in this library. Each file should start with an underscore      |
 | `deps`    | Dependencies for the `src`. Each dependency is a `sass_library`                     |
+
+### multi_sass_binary
+
+```py
+multi_sass_binary(name, srcs=[], output_style="compressed", sourcemap=True)
+```
+
+`multi_sass_binary` compiles a list of Sass files and outputs the corresponding
+CSS files and optional sourcemaps. Output is omitted for filenames that start
+with underscore "_".
```

Comment:
do Angular apps already follow the convention of using underscore-prefixed names for partials that aren't meant to produce css binaries?

Reply:
by default, the stylesheets generated by CLI are not prefixed by underscore.